domains/itsdemongamez.json

```diff
@@ -1,11 +1,11 @@
 {
     "owner": {
-        "username": "ShadowPlayzYT22",
-        "email": "peterwyneg@gmail.com"
+       "username": "ShadowPlayzYT22",
+       "email": "peterwyneg@gmail.com",
+       "discord": "1038747272701292584"
     },
+
     "record": {
-        "A": ["217.174.245.249"],
-        "MX": ["hosts.is-a.dev"],
-        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all"
+        "CNAME": "Demon-Gamez-website.github.io"
```

Comment:
404 CNAME record. Please provide valid record.

Reply:
That is the correct cname, The cname should be the the github repostory name (100% sure), This cname worked before i changed the records to txt and stuff. Please correct me if im wrong. And by the way, I think you cannot reach websites using the cname.

Reply:
It isn't. Read the docs!

Reply:
hmmm, Okay